addons/website/models/website_controller_page.py

```diff
@@ -12,18 +14,26 @@ class WebsiteControllerPage(models.Model):
         'website.searchable.mixin',
     ]
     _description = 'Model Page'
-    _order = 'website_id'
+    _order = 'website_id, id DESC'
+    _sql_constraints = [
+        ('unique_name_slugified', 'UNIQUE(page_type, name_slugified)', 'url should be unique')
+    ]
```

Comment:
Is this not too constraining in multiwebsite ? Maybe `website_id` should also be included in the unique key ?

Reply:
the condition will be more complicated then. The point is to have one determined access point.
If all pages have a website_id, then, yeah, fine, but if some don't they could collide with the ones which do for one given website.